randovania/gui/ui_files/multiplayer_session.ui

```diff
@@ -146,14 +146,20 @@
            <item row="2" column="0">
             <widget class="QCheckBox" name="everyone_can_claim_check">
              <property name="text">
-              <string>Everyone can claim worlds</string>
+              <string>Everyone Can Claim Unclaimed Worlds</string>
```

Comment:
Is there are reason why this is capitalized now? I seriously have no idea what exactly are our own rules for it. "World's" in this context is a randovania term and should be fine but dunno about the rest. (It is often all capitalized, no?)